CONTRIBUTING.md

```diff
@@ -80,6 +80,8 @@ git checkout -b fix/some-bug
 git checkout -b remove/some-file
 ```
 
+If your branch is planned to fix an open issue, postfix your branch name with the issue number like `fix/some-bug-#123`.
```

Comment:
I see that this was done in this PR but Waffle didn't associate it to the corresponding issue correctly. It may be related to the lack of "fixes #639" thing mentioned above?

Reply:
ah, looks like you do need both: https://help.waffle.io/automatic-work-tracking/auto-work-tracking-basics/recommended-workflow-using-pull-requests-automatic-work-tracking